src/router.ts

```diff
@@ -111,6 +143,23 @@ export class Router {
   */
   viewPortDefaults: any = {};
 
+  /**@internal */
+  catchAllHandler: (instruction: NavigationInstruction) => NavigationInstruction | Promise<NavigationInstruction>;
+  /**@internal */
+  fallbackRoute: string;
```

Comment:
not internal

Reply:
Same for this, it's not in the router in our current `aurelia-router.d.ts`

Reply:
I'd vote for it to be like it is atm. We can always change it to public if necessary.